listenbrainz/db/tests/test_feedback.py

```diff
@@ -197,9 +191,8 @@ def test_get_feedback_for_user_with_metadata(self):
         self.assertEqual(result[0].score, self.sample_feedback_with_metadata[0]["score"])
         self.assertEqual(result[0].track_metadata["artist_name"], "Portishead")
         self.assertEqual(result[0].track_metadata["track_name"], "Strangers")
-        self.assertEqual(result[0].track_metadata["additional_info"]["recording_mbid"], "076255b4-1575-11ec-ac84-135bf6a670e3")
-        self.assertEqual(result[0].track_metadata["additional_info"]["release_mbid"], "1fd178b4-1575-11ec-b98a-d72392cd8c97")
-        self.assertEqual(result[0].track_metadata["additional_info"]["artist_msid"], self.saved_artist_msid)
+        self.assertEqual(result[0].track_metadata["mbid_mapping"]["recording_mbid"], "076255b4-1575-11ec-ac84-135bf6a670e3")
```

Comment:
This made me realise that currently feedback endpint serves mapped mbids inside additional_info key whereas pinned recordings and listens do that in mbid_mapping. The frontend checks both so no issues there. Since this is a MSID endpoint, I doubt there are any 3rd party consumers of this API. So I am in favour of making the switch to using the mbid_mapping key here right away.